src/services/filters.js

```diff
@@ -21,11 +21,140 @@ goog.require('ngeo');
  * @ngname ngeoScalify
  */
 ngeo.Scalify = function($filter) {
-  var number = $filter('number');
+  var numberFilter = $filter('number');
   return function(scale) {
-    var text = number(scale, 0);
+    var text = numberFilter(scale, 0);
     return text ? '1\u00a0:\u00a0' + text : '';
   };
 };
 
 ngeo.module.filter('ngeoScalify', ngeo.Scalify);
+
+/**
+ * Format a couple of numbers as number coordinates.
+ *
+ * Example without parameters:
+ *
+ *      <p>{{[7.1234, 46.9876] | ngeoNumberCoordinates}}</p>
+ *      <!-- will Become 7 47 -->
+ *
+ * Example with defined fractionDigits and template:
+ *
+ *      <p>{{[7.1234, 46.9876] | ngeoNumberCoordinates:2:co {x} E; {y} N}}</p>
+ *      <!-- will Become co 7.12 E; 46.99 N -->
+ *
+ * Example without fractionDigits but with defined template and localize:
+ *
+ *      <!-- With en-US localization -->
+ *      <p>{{[2600000, 1600000] | ngeoNumberCoordinates::{x}, {y}:true}}</p>
+ *      <!-- will Become 2,600,000, 1,600,000 -->
+ *      <br/>
+ *      <!-- With fr-CH localization -->
+ *      <p>{{[2600000, 1600000] | ngeoNumberCoordinates::{x}, {y}:true}}</p>
+ *      <!-- will Become 2'600'000, 1'600'000 -->
+ *      <br/>
+ *      <!-- Without localization (or localize to false) -->
+ *      <p>{{[2600000, 1600000] | ngeoNumberCoordinates::{x}, {y}}}</p>
+ *      <!-- will Become 2'600'000, 1'600'000 -->
+ *
+ * @param {angular.$filter} $filter Angular filter
+ * @return {function(ol.Coordinate, number=, string=): string} A
+ *     function to format numbers into coordinates string.
+ * @ngInject
+ * @ngdoc filter
+ * @ngname ngeoNumberCoordinates
+ */
+ngeo.NumberCoordinates = function($filter) {
+  /**
+   * @param {ol.Coordinate} coordinates Array of two numbers.
+   * @param {number=} opt_fractionDigits number of digit, Same as
+   *     {@link ol.coordinate.format} opt_fractionDigits parameters.
+   * @param {string=} opt_template Optional template. Default to '{x} {y}'.
+   *     Where "{x}" will be replaced by the first coordinate and "{y}" by the
+   *     second one. Note: Use a html entity to use the semicolon symbole
+   *     into a template.
+   * @param {boolean=} opt_localize Optional boolean to format number on
+   *     the current local {@link Angular.filter.number}. By default numbers use
+   *     "." as the decimal separator and include "'" group separators after
+   *     each third digit).
+   * @return {string} East north formated coordinates.
+   */
+  var filterFn = function(coordinates, opt_fractionDigits, opt_template,
+      opt_localize) {
+    var template = opt_template ? opt_template : '{x} {y}';
+    var x = coordinates[0];
+    var y = coordinates[1];
+    if (opt_localize) {
```

Comment:
=> `if (opt_localize !== false) {` ?
I think that's should be true by default...